src/index.html

```diff
@@ -1,13 +1,282 @@
+                alt="one ear piece"
+                class="categories__image"
+              >
+            </a>
+            <a href="#" class="categories__products-4">
+              <img
+                src="./images/categories/ear-stick.png"
+                alt="ear stick"
+                class="categories__image categories__image--img-after-tablet"
+              >
+              <img
+                src="./images/categories/ear-stick-mobile.svg"
+                alt="ear stick"
+                class="categories__image categories__image--img-before-tablet"
+              >
+            </a>
+          </div>
+          <h3 class="categories__description">Audio</h3>
+        </article>
+        <article class="categories__section">
+          <div class="categories__products">
+            <a href="#" class="categories__products-5">
+              <img
+                src="./images/categories/phone.png"
+                alt="phone"
+                class="categories__image categories__image--img-after-tablet"
+              >
+              <img
+                src="./images/categories/phone-mobile.svg"
+                alt="phone"
+                class="categories__image categories__image--img-before-tablet"
+              >
+            </a>
+            <a href="#" class="categories__products-6">
+              <img
+                src="./images/categories/charger.svg"
+                alt="charger"
+                class="categories__image"
+              >
+            </a>
+          </div>
+          <h3 class="categories__description">Accessories</h3>
+        </article>
+        <a href="#" class="categories__view-all">View all</a>
+      </section>
+
+      <section class="about-us" id="about-us">
+        <img
+          src="./images/about-us/bg-before-about-us.svg"
+          alt="background"
+          class="about-us__bg"
+        >
+        <div class="about-us__container">
+          <h2 class="section-title about-us__title">About us</h2>
+          <p class="about-us__description">We’re a London-based tech company on a mission to remove barriers between people and technology.
+            <br>
+            <br>
+            To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+            <br>
+            No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+            <br>
+            <br>
+            Because beautiful tech shouldn’t be complicated.
+          </p>
+
+        </div>
+      </section>
+
+      <section class="contact" id="contact">
+        <div class="contact-container">
+          <div class="contact-container__form">
+            <h2 class="section-title">Contact us</h2>
+            <form
+              action="http://localhost:1234/api"
+              method="post"
+              class="contact__form"
+            >
```

Comment:
The page shouldn't reload upon form submission [Checklist #15](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=The%20page%20shouldn%27t%20be%20reloaded%20on%20form%20submit%2C%20but%20the%20fields%20should%20be%20cleared.)